specs/utility/time.table

```diff
@@ -9,11 +9,8 @@ schema([
     Column("minutes", INTEGER, "Current minutes in the system"),
     Column("seconds", INTEGER, "Current seconds in the system"),
     Column("timezone", TEXT, "Current timezone in the system"),
```

Comment:
This is now hardcoded, not the system timezone. I'd probably change the desc of all of these, they're not system time. They're in UTC